### DISCUSSION TITLE
Do we have any plan to support react 18?

### DISCUSSION THREAD

**Discussion by frank3en** [trendmicro-frontend/tonic-ui#591] [General]
Hi everyone, as react18 has been released for serveral months, do we have any plan to support the new version of it? @cheton 
<img width="888" alt="image" src="https://user-images.githubusercontent.com/79500516/170406848-c91a703b-c591-4343-80a6-6023fe07f3cd.png">

**Comment by cheton** [comment]
Added in `@tonic-ui/react` 1.7.0
https://github.com/trendmicro-frontend/tonic-ui/releases/tag/%40tonic-ui%2Freact%401.7.0
